### PR TITLE
:sparkles: TK-3950 Test cases for Application YAMLs & Trilio YAMLs

### PR DESCRIPTION
**Pull request by vishutrilio**
Changes Done:
* Added positive and negative scenarios for resource-metadata & trilio-resources CLI

### CODE REVIEW

File: tests/target-browser/target_browser_test.go [hunk 1]
```diff
@@ -1624,6 +1631,278 @@ var _ = Describe("Target Browser Tests", func() {
...
+			})
+
+			AfterEach(func() {
+				if isLast {
+					// delete target & remove all files & directories created for this Context - only once After all It in this context
+					deleteTarget(false)
+					for _, backupPlans := range backupPlanUIDs {
+						_, err := shell.RmRf(filepath.Join(TargetLocation, backupPlans))
+						Expect(err).To(BeNil())
+					}
```
Comment (pankaj9310) [inline tests/target-browser/target_browser_test.go:1866]: @vishutrilio Pls use func `removeBackupPlanDir`

File: tests/target-browser/target_browser_test.go [hunk 2]
```diff
@@ -1624,6 +1631,278 @@ var _ = Describe("Target Browser Tests", func() {
...
```
Comment (pankaj9310) [inline tests/target-browser/target_browser_test.go:1788]: @vishutrilio pls use this `removeBackupPlanDir`

File: tests/target-browser/target_browser_test.go (hunk 1 above)
Comment (vishutrilio) [reply tests/target-browser/target_browser_test.go]: Done

File: tests/target-browser/target_browser_test.go (hunk 2 above)
Comment (vishutrilio) [reply tests/target-browser/target_browser_test.go]: Done

**Review by madhurtrilio — APPROVED** [review]

**Pull request merged**